File: 75-sort-colors/75-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        isSorted=False
        while not isSorted:
            isSorted=True
            for i in range(len(nums)-1):
                if nums[i]>nums[i+1]:
                    self.swap(i,i+1, nums)
                    isSorted=False
        
        return nums
    
    def swap(self, num1, num2, array):
        array[num1], array[num2] = array[num2], array[num1]

File: 75-sort-colors/75-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        boolean isSorted=false;
        int count=0;
        
        while(!isSorted){
            isSorted=true;
            
            for(int i=0; i<nums.length-1-count;i++){
                if(nums[i]>nums[i+1]){
                    swap(i,i+1, nums);
                }
                isSorted=false;
            }
            count++;
        }
        
    }
    
    public static void swap(int nums1, int nums2, int[] array){
        int temp = array[nums1];
        array[nums1]= array[nums2];
        array[nums2]= temp;
    }
}